<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTKT</class>
 <widget class="QWidget" name="CTKT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>发自内心的话</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>真的很稀饭你，
愿意做我女朋友吗？</string>
   </property>
  </widget>
  <widget class="QPushButton" name="agreeButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>同意</string>
   </property>
   <property name="icon">
    <iconset theme=":/hart.ico"/>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="disagreeButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>不同意</string>
   </property>
  </widget>
  <widget class="QLabel" name="pictureMain">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>181</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>://20200726180125.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>pictureMain</zorder>
  <zorder>label</zorder>
  <zorder>agreeButton</zorder>
  <zorder>disagreeButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
